integration_test/messageDataParser.test.ts

```diff
@@ -0,0 +1,74 @@
+    const retryableData =
+      '0x000000000000000000000000467194771DAE2967AEF3ECBEDD3BF9A310C76C650000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000030346F1C785E00000000000000000000000000000000000000000000000000000053280CF1490000000000000000000000007F869DC59A96E798E759030B3C39398BA584F0870000000000000000000000007F869DC59A96E798E759030B3C39398BA584F08700000000000000000000000000000000000000000000000000000000000210F100000000000000000000000000000000000000000000000000000000172C586500000000000000000000000000000000000000000000000000000000000001442E567B360000000000000000000000006B175474E89094C44DA98B954EEDEAC495271D0F0000000000000000000000007F869DC59A96E798E759030B3C39398BA584F0870000000000000000000000007F869DC59A96E798E759030B3C39398BA584F08700000000000000000000000000000000000000000000003871022F1082344C7700000000000000000000000000000000000000000000000000000000000000A000000000000000000000000000000000000000000000000000000000000000800000000000000000000000000000000000000000000000000000000000000040000000000000000000000000000000000000000000000000000000000000006000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000'
+
+    const res = messageDataParser.parse(retryableData)
+
+    expect(res.callValueRefundAddress).to.eq(
+      '0x7F869dC59A96e798e759030b3c39398ba584F087'
+    )
+    expect(res.data).to.eq(
+      '0x2E567B360000000000000000000000006B175474E89094C44DA98B954EEDEAC495271D0F0000000000000000000000007F869DC59A96E798E759030B3C39398BA584F0870000000000000000000000007F869DC59A96E798E759030B3C39398BA584F08700000000000000000000000000000000000000000000003871022F1082344C7700000000000000000000000000000000000000000000000000000000000000A000000000000000000000000000000000000000000000000000000000000000800000000000000000000000000000000000000000000000000000000000000040000000000000000000000000000000000000000000000000000000000000006000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000'
+    )
+    expect(res.destAddress).to.eq('0x467194771dAe2967Aef3ECbEDD3Bf9a310C76C65')
+    expect(res.excessFeeRefundAddress).to.eq(
+      '0x7F869dC59A96e798e759030b3c39398ba584F087'
+    )
+    expect(res.gasLimit.eq(BigNumber.from('0x0210f1')), 'incorrect gas limit')
+      .to.be.true
+    expect(
+      res.l1Value.eq(BigNumber.from('0x30346f1c785e')),
+      'incorrect l1 value'
+    ).to.be.true
+    expect(res.l2CallValue.eq(BigNumber.from(0)), 'incorrect l2 call value').to
+      .be.true
+    expect(
+      res.maxFeePerGas.eq(BigNumber.from('0x172c5865')),
+      'incorrect max fee per gas'
+    ).to.be.true
+    expect(
+      res.maxSubmissionFee.eq(BigNumber.from('0x53280cf149')),
+      'incorrect max submission fee'
+    ).to.be.true
+  })
+
+  it('does parse eth deposit message', async () => {
```

Comment:
We should update this to parse nitro's Eth deposit message

Reply:
Yeh, I left the old one there since that's still a valid kind of l1 to l2 message. But we can add some tests for the actual eth deposit message soon too.

I'll update the comment here